include/libopencm3/swm050/gpio.h

```diff
@@ -47,52 +44,74 @@
 #define GPIO7				(1 << 7)
 #define GPIO8				(1 << 8)
 #define GPIO9				(1 << 9)
-#define GPIO_ALL			0x3ff
-/**@}*/
+#define GPIO_ALL			(0x3ff)
+/*@}*/
 
 /* GPIO direction definitions */
 /** @defgroup gpio_dir GPIO Pin Direction
 @{*/
-#define GPIO_INPUT			0x0
-#define GPIO_OUTPUT			0x1
-/**@}*/
+#define GPIO_INPUT			(0x0)
+#define GPIO_OUTPUT			(0x1)
+/*@}*/
```

Comment:
drop all the gratuitous changes in this file please.  No need for bracketing, and no need for dropping doxygen comments even if it happens to work sometimes with some versions of doxygen

Reply:
Fixed.